src/pages/Api.js

```diff
@@ -0,0 +1,116 @@
+export default class Api {
+  constructor(options) {
+    this.baseUrl = options.baseUrl;
+    this.headers = options.headers;
+  }
+
```

Comment:
The `Api.js` file should be in the `components` folder, according to the checklist